dockerfile-image-update/src/test/java/com/salesforce/dockerfileimageupdate/subcommands/impl/ParentTest.java

```diff
@@ -109,6 +109,49 @@ public void testForkRepositoriesFound() throws Exception {
         assertEquals(repoMap.size(), 3);
     }
 
+    @Test
+    public void testForkRepositoriesFound_unableToforkRepo() throws Exception {
```

Comment:
Same. What are we testing? What is the expected result here?

Reply:
Added comments.